MediaBrowser.Common/Json/Converters/JsonOmdbNotAvailableStringConverter.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace MediaBrowser.Common.Json.Converters
+{
+    /// <summary>
+    /// Converts a string <c>N/A</c> to <c>string.Empty</c>.
+    /// </summary>
+    public class JsonOmdbNotAvailableStringConverter : JsonConverter<string>
+    {
+        /// <inheritdoc />
+        public override string Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+        {
+            if (reader.TokenType == JsonTokenType.String)
+            {
+                var str = reader.GetString();
+                if (str != null && str.Equals("N/A", StringComparison.OrdinalIgnoreCase))
+                {
+                    return null;
+                }
+            }
+
+            return JsonSerializer.Deserialize<string>(ref reader, options);
```

Comment:
Can't we return `str` inside of the if statement if it isn't `N/A`? @crobibero

Reply:
Yes we need to return `str`, else we get a stack overflow